BedrockServer.h

```diff
@@ -147,6 +147,7 @@ class BedrockServer : public SQLiteServer {
     static constexpr auto STATUS_HANDLING_COMMANDS = "GET /status/handlingCommands HTTP/1.1";
     static constexpr auto STATUS_PING              = "Ping";
     static constexpr auto STATUS_STATUS            = "Status";
+    static constexpr auto STATUS_WHITELIST         = "SetCommandWhitelist";
```

Comment:
Ug, I hate string constants.  Why are these prefixed `STATUS_`?  If we're going to have them can we name them:

* `PING = "Ping"`
* `STATUS = "Status"`
* `SET_MULTIWRITE_WHITELIST = "SetMultiwriteWhitelist"`

Regardless, I'd vote that we just remove these constants entirely.